mixer/pkg/runtime2/handler/factory.go

```diff
@@ -0,0 +1,203 @@
+// Copyright 2018 Istio Authors
+//
```

Comment:
Is it possible to see this file as branched from original factory.go ?

Reply:
other files too in this PR. Thanks, it will help to see the delta, but I am not sure if it is doable.